articles/firewall-manager/migrate-to-policy.md

```diff
@@ -25,11 +25,11 @@ Modify the following script to migrate your firewall configuration.
 
 ```azurepowershell
 #Input params to be modified as needed
-$FirewallName = "AZFW"
-$ResourceGroupName = "AzFWMigrateRG"
-$PolicyName = "fwp9"
-$Location = "WestUS"
-
+$FirewallResourceGroup = "fw-rg"
```

Comment:
Can we please have the resource group names starting with an uppercase

Reply:
Added with capital letters.